gradle/apps/rest/src/main/java/com/liferay/blade/samples/rest/UsersRestService.java

```diff
@@ -60,6 +61,13 @@ public String getUsers() {
 		return result.toString();
 	}
 
+	public void showLog() {
```

Comment:
No one calls show log.  We should just add the logging in the component rest method or in the activate so they can see it activating.